Automattic/woocommerce-payments
# Prevent platform checkout iframe appear when go back

Fixes #4065

#### Changes proposed in this Pull Request

* Fix a bug that always show the platform checkout when go back from it.









#### Testing instructions





* Enable platform checkout.
* Go to checkout page and fill out you email address.
* Once redirected, go back with browser button.
* It should not show the platform checkout iframe.
* Remove a letter from your email address and add it again.
* Once redirected, click the back button on platform checkout.
* It should not show the platform checkout iframe.

-------------------


**Post merge**
